Production/govuk_ios/Extensions/Analytics/AppEvent+Convenience.swift

```diff
@@ -30,17 +30,32 @@ extension AppEvent {
         )
     }
 
+    static func searchItemNavigation(title: String,
+                                     url: URL,
+                                     external: Bool) -> AppEvent {
+        navigation(
+            text: title,
+            type: "SearchResult",
+            external: external,
+            extraParams: ["url": url.absoluteString]
+        )
+    }
+
     static func navigation(text: String,
                            type: String,
-                           external: Bool) -> AppEvent {
-        .init(
+                           external: Bool,
+                           extraParams: [String: Any?] = [:]) -> AppEvent {
```

Comment:
This is soo minor, but can you rename this to `additionalParams` or `additionalParameters` I will explain why in slack